apps/directory/src/components/report-components/FactsTable.vue

```diff
@@ -450,6 +450,10 @@ export default {
                       fact[column],
                     ];
                   }
+                } else {
+                  if (column == "number_of_samples") {
```

Comment:
Please use `else if` and `===` as comparator .

Reply:
made the change. This change has to be propagated to the 3970 issue. I made the change also on that branch (branch issue3970) if you want to merge directly only that branch with the master.